webservices/legal_docs/advisory_opinions.py

```diff
@@ -58,18 +58,24 @@
     WHERE ao_id = %s
 """
 
-STATUTE_CITATION_REGEX = re.compile(r"(?P<title>\d+)\s+U.S.C.\s+§*(?P<section>\d+).*\.?")
-REGULATION_CITATION_REGEX = re.compile(r"(?P<title>\d+)\s+CFR\s+§*(?P<part>\d+)\.(?P<section>\d+)")
-AO_CITATION_REGEX = re.compile(r"\b(?P<year>\d{4,4})-(?P<serial_no>\d+)\b")
+STATUTE_CITATION_REGEX = re.compile(
+    r"(?P<title>\d+)\s+U.S.C.\s+§*\s*(?P<section>\d+).*\.?")
+
+REGULATION_CITATION_REGEX = re.compile(
+    r"(?P<title>\d+)\s+C\.*F\.*R\.*\s+§*\s*(?P<part>\d+)\.(?P<section>\d+)")
```

Comment:
Just noting that you have the `*` character here with `C.F.R.` and including the `*` modifies the preceding character and means "0 or more matches" - if this is intended, and looking at the test cases above it seems to be the case since you can have `CFR` or `C.F.R` match, then this is good!  I just wanted to make sure this was the desired outcome. :-)

Reply:
@ccostino yes, thank you! That's my understanding of the desired behavior.